test/test_data/BUILD

```
@@ -34,6 +34,34 @@ universal_binary(
 
 starlark_apple_binary(
     name = "apple_binary",
```

Perhaps rename this in a follow up to be consistent?
```suggestion
    name = "macos_binary",
```